src/Microsoft.Azure.SignalR.Management/RestHubLifetimeManager.cs

```diff
@@ -263,6 +263,78 @@ private static void ValidateUserIdAndGroupName(string userId, string groupName)
             }
         }
 
+        public async Task<bool> CheckIfConnectionExistsAsync(string connectionId, CancellationToken cancellationToken = default)
+        {
+            if (string.IsNullOrEmpty(connectionId))
+            {
+                throw new ArgumentException(NullOrEmptyStringErrorMessage, nameof(connectionId));
+            }
+            var exists = false;
+            var api = await _restApiProvider.GetCheckConnectionExistsEndpointAsync(_appName, _hubName, connectionId);
+            await _restClient.SendAsync(api, HttpMethod.Head, _productInfo, handleExpectedResponse: response =>
+            {
+                switch (response.StatusCode)
+                {
+                    case HttpStatusCode.OK:
+                        exists = true;
+                        return true;
+                    case HttpStatusCode.NotFound:
+                        return true;
```

Comment:
return false?

Reply:
The return value indicates whether to throw an exception. The variable used to indicate existing is `exists`.

Reply:
Intend to refactor `RestClient` later

Reply:
I see